code/src/java/pcgen/core/PlayerCharacter.java

```diff
@@ -9515,9 +9631,9 @@ public void setStat(PCStat stat, int value)
 		statValueFacet.set(id, stat, value);
 	}
 
-	public Integer getStat(PCStat stat)
```

Comment:
To avoid a double-boxing operation.  The old version was guaranteed to return an Integer from the get, so the return of Integer guaranteed no boxing operation.  The new version returns a Number and calls intValue, so it is guaranteed to have an int, so we return int to avoid boxing.  In both cases, we return natively what we know we have, so that if someone else wants exactly what we have, we don't box and then unbox.